### PR TITLE
Updated ignore files

### PR DESCRIPTION
**Pull request by Aju100**
I have updated some ignored file which are required for jekyll based static website.

### CODE REVIEW

**Comment by Aju100** [comment]
@coolbrg . I knew through internet. can u accept these PR.

**Comment by Aju100** [comment]
later on, I will do some few contributions

**Comment by coolbrg** [comment]
> @coolbrg . I knew through internet. can u accept these PR.

@Aju100 Apology here but I can't merge this PR as it contains some unnecessary information in context of the project. 
Example: So far I have only occurred `*.DS_Store` in my all projects I have done so far in macOS and never encountered `.AppleDouble` or others.

It's good to have awareness about others but it's unnecessary to have it. 
Check https://github.com/rails/rails/blob/master/.gitignore or https://github.com/django/django/blob/master/.gitignore projects.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,81 @@
+### Jekyll ###
 _site
 .sass-cache
 .jekyll-metadata
+Gemfile.lock
```
Comment (coolbrg) [inline .gitignore]: This is valid.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+
+### Linux ###
+*~
+.fuse_hidden*
+.Trash-*
+.nfs*
+
+
+### MacOS ###
+*.DS_Store
```
Comment (coolbrg) [inline .gitignore]: This `*.DS_Store` is valid.

Comment (coolbrg) [reply .gitignore]: Except for the commented ones, I recommend deleting others

Comment (Aju100) [reply .gitignore]: sure

**Comment by Aju100** [comment]
@coolbrg  I have deleted the rest of them of it. You can review it.

**Pull request merged**